test/tests/serial/user.bats

```diff
@@ -0,0 +1,42 @@
+load "${REPOSITORY_ROOT}/test/helper/common"
+load "${REPOSITORY_ROOT}/test/helper/setup"
+
+BATS_TEST_NAME_PREFIX='[User] '
+CONTAINER_NAME='dms-test_User_change_uid'
+
+function setup_file() {
+  _init_with_defaults
+
+  local CUSTOM_SETUP_ARGUMENTS=(
+    --env ENABLE_POP3=1
+    --env PERMIT_DOCKER=container
+    --env UID_DOCKER=10000
+  )
+
+  _common_container_setup 'CUSTOM_SETUP_ARGUMENTS'
+  _wait_for_smtp_port_in_container users
```

Comment:
The helper method has an implicit default for `${CONTAINER_NAME}`, you can remove being explicit here.

```suggestion
  _wait_for_smtp_port_in_container
```

---

In this case the value given isn't even correctly mapped to a container name assigned. But the internal helper method is detecting the CONTAINER_NAME and using that.
https://github.com/docker-mailserver/docker-mailserver/blob/8c0cfa0836bcf20e78a2d5d1ae02270004ee0aed/test/helper/common.bash#L246-L252

https://github.com/docker-mailserver/docker-mailserver/blob/8c0cfa0836bcf20e78a2d5d1ae02270004ee0aed/test/helper/common.bash#L53C1-L65

https://github.com/docker-mailserver/docker-mailserver/blob/8c0cfa0836bcf20e78a2d5d1ae02270004ee0aed/test/helper/common.bash#L232-L244